CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing to Eclipse sensiNact
+
+Eclipse sensiNact is an open source project hosted by the Eclipse Foundation. Contributions are accepted from across the community.
+
+## Issues
+
+If you're having a problem with Eclipse sensiNact then our first recommendation would be to look at [the documentation](https://eclipse-sensinact.readthedocs.io/en/latest/).
+
+If you're still having problems then search the [issue tracker](https://github.com/eclipse-sensinact/org.eclipse.sensinact.gateway/issues) for a relevant issue or create a new one.
+
+## Contributing code
+
+If you want to contribute fixes or enhancements to Eclipse sensiNact then you can do so using a *triangular workflow*.
+
+* Fork the repository in GitHub and make changes in your fork.
+  - Make sure that your changes are small and easy to understand. Separate changes into multiple commits where that makes sense
+  - Commit messages should be detailed and descriptive
+  - Include automated tests for any bug fixes and new code
+  - Each commit should try to do one thing, and the code should build cleanly with each commit
+* Submit a pull request to the Eclipse sensiNact repository
+  - Make sure that the pull request passes all of the CI checks
+
```

Comment:
Maybe add a line to request for tests in association to pull requests

Reply:
Would this cover it? 

https://github.com/eclipse-sensinact/org.eclipse.sensinact.gateway/pull/495/files#r1843515972